scripts/site-config.js

```diff
@@ -292,13 +292,15 @@ export async function getQuickLinkConfig() {
       result.push({
         title: row.Title,
         page: row.Page,
+        hide: row.Hide,
       });
     } else if (row.Title && row.Page && row.Group) {
```

Comment:
both if and else block has same code except the nested if block. can you try to merge both conditions blocks?

Reply:
The 2 if statements can be consolidated, I don't think.
1st statement is for when the group is empty
2nd statement is for when the group is not empty

Reply:
Refactored the if logic code